packages/plugins/src/hardhat/tasks.ts

```diff
@@ -1086,7 +1064,7 @@ task(TASK_TEST)
           if (!noCompile) {
             await cleanThenCompile(hre)
           }
-          await deployAllChugSplashConfigs(hre, !show, '', true)
+          await deployAllChugSplashConfigs(hre, !show, '', true, true)
```

Comment:
Since the test task only ever deploys contracts when run against the hardhat network, I just removed the confirm flag like the `node` task.